src/auth/auth.service.ts

```
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/prisma/prisma.service';
+
+@Injectable()
+export class AuthService {
+  constructor(private prisma: PrismaService) {}
+
+  //handle new user register
+  async register(){
```

ESLint

[error] Insert `·`

(prettier/prettier)